packages/metro-config/transformer.js

```diff
@@ -0,0 +1,14 @@
+const transformer = require('metro-react-native-babel-transformer');
+
+exports.getCacheKey = transformer.getCacheKey;
+
+exports.transform = function({ filename, options, src, plugins }) {
+  return transformer.transform({
+    filename,
+    options: Object.assign({}, options, {
+      extendsBabelConfigPath: require.resolve('babel-preset-expo'),
```

Comment:
nice

Reply:
The transformer is copy pasted from your Metro PR, so thank you 😄